ast.hpp

```diff
@@ -1667,9 +1667,6 @@ namespace Sass {
         if (has_rest_parameter_) {
           error("functions and mixins cannot have more than one variable-length parameter", p->pstate());
         }
-        if (has_optional_parameters_) {
```

Comment:
IMO it should be spot on now. The test I removed is obviously wrong, since rest parameters can come after a parameter with optional (default) value (as you can see `has_optional_parameters_` is set when we had a preceding `default_value`. Or can you come up with a sample that doesn't behave correctly?